examples/kernel_approximation/plot_scalable_poly_kernels.py

```diff
@@ -98,6 +114,9 @@
 # (`n_runs` = 1) in this example, in practice one should repeat the experiment several
 # times to compensate for the stochastic nature of :class:`PolynomialCountSketch`.
 
+from sklearn.pipeline import Pipeline
```

Comment:
Can we also use `make_pipeline` here and save some lines?

Reply:
This is a good point! I have replaced the use of `Pipeline` with `make_pipeline`, as you suggest.

Reply:
You are still using Pipeline, maybe you have not pushed your commit?

Reply:
Ah, my mistake. Thank you for letting me know! The change should be reflected correctly now.